k8s/overlays/test/fixture.json

```diff
@@ -1 +1 @@
-{"resourceSpans":[{"resource":{"attributes":[{"key":"host.name","value":{"stringValue":"foo.bar.host"}},{"key":"process.command_args","value":{"arrayValue":{"values":[{"stringValue":"./main"}]}}},{"key":"process.executable.name","value":{"stringValue":"main"}},{"key":"process.executable.path","value":{"stringValue":"/go/src/github.com/open-telemetry/opentelemetry-collector-contrib/examples/demo/server/main"}},{"key":"process.owner","value":{"stringValue":"mikedame"}},{"key":"process.pid","value":{"intValue":"2634978"}},{"key":"process.runtime.description","value":{"stringValue":"go version go1.19-pre4 cl/455575533 +12f49fe0ed linux/amd64"}},{"key":"process.runtime.name","value":{"stringValue":"go"}},{"key":"process.runtime.version","value":{"stringValue":"go1.19-pre4 cl/455575533 +12f49fe0ed"}},{"key":"service.name","value":{"stringValue":"demo-server"}},{"key":"service.namespace","value":{"stringValue":""}},{"key":"telemetry.sdk.language","value":{"stringValue":"go"}},{"key":"telemetry.sdk.name","value":{"stringValue":"opentelemetry"}},{"key":"telemetry.sdk.version","value":{"stringValue":"1.8.0"}}]},"scopeSpans":[{"scope":{"name":"go.opentelemetry.io/contrib/instrumentation/net/http/otelhttp","version":"semver:0.33.0"},"spans":[{"traceId":"2499d13df7b35ae93dfa3d6ddc01da74","spanId":"84f9e8929bb1cd5b","parentSpanId":"a514c25ace3f0a5a","name":"/hello","kind":"SPAN_KIND_SERVER","startTimeUnixNano":"1658413463217165380","endTimeUnixNano":"1658413463225568609","attributes":[{"key":"net.transport","value":{"stringValue":"ip_tcp"}},{"key":"net.peer.ip","value":{"stringValue":"127.0.0.1"}},{"key":"net.peer.port","value":{"intValue":"32878"}},{"key":"net.host.ip","value":{"stringValue":"0.0.0.0"}},{"key":"net.host.port","value":{"intValue":"7080"}},{"key":"http.method","value":{"stringValue":"GET"}},{"key":"http.target","value":{"stringValue":"/hello"}},{"key":"http.server_name","value":{"stringValue":"/hello"}},{"key":"http.user_agent","value":{"stringValue":"Go-http-client/1.1"}},{"key":"http.scheme","value":{"stringValue":"http"}},{"key":"http.host","value":{"stringValue":"0.0.0.0:7080"}},{"key":"http.flavor","value":{"stringValue":"1.1"}},{"key":"server-attribute","value":{"stringValue":"foo"}},{"key":"baggage key:method","value":{"stringValue":"repl"}},{"key":"baggage key:client","value":{"stringValue":"cli"}},{"key":"http.wrote_bytes","value":{"intValue":"11"}},{"key":"http.status_code","value":{"intValue":"200"}}],"status":{}}]}],"schemaUrl":"https://opentelemetry.io/schemas/1.10.0"}]}
```

Comment:
any idea why the change here? Looks like `host.name` got dropped, not sure if there's any other changes

Reply:
when I run `make generate` on `main` I get the same result. Maybe when we checked in the change, we forgot to run `make generate` one last time?